### ISSUE TITLE
Oracle error  ORA-00933: SQL command not properly ended thrown during CI tests

### ISSUE THREAD

**Issue by t-abdul-basser** [OHDSI/Achilles#287]
### Expected behavior

Travis build tests pass

### Actual behavior

Oracle test fails on master

### Steps to reproduce behavior

Run Travis build on master.

See [build #399](https://travis-ci.org/OHDSI/Achilles/builds/403326221).
```
java.sql.SQLSyntaxErrorException: ORA-00933: SQL command not properly ended
SQL:
insert into [secure].ACHILLES_results_derived (statistic_value,measure_id)    
SELECT CAST(100.0*count_value/statistic.total_pts AS FLOAT) as statistic_value,
   CAST(CONCAT('ach_', CONCAT(CAST(analysis_id as VARCHAR(10)), ':Percentage')) AS VARCHAR(100)) as measure_id
  FROM [secure].achilles_results 
	cross join (SELECT count_value as total_pts FROM [secure].achilles_results r   WHERE analysis_id =1 ) as statistic
    WHERE analysis_id in (2000,2001,2002,2003)
```

**Comment by vojtechhuser** [comment]
the semicolon is there. See here https://github.com/OHDSI/Achilles/blob/e7c1ea15c6dc75eab9906be9097c732da7f256e4/inst/sql/sql_server/AchillesHeel_v5.sql#L145

**Comment by t-abdul-basser** [comment]
Thanks @vojtechhuser. Finishing fix now.

**Comment by t-abdul-basser** [comment]
Likely related to OHDSI/SqlRender#133. Likely to be fixed with recent merge of OHDSI/SqlRender#135.

**Comment by chrisknoll** [comment]
No, that SqlRender is to deal with ```FROM x AS y```.  Could be just a missing semicolin in this case.
